README.rst

```diff
@@ -163,6 +163,41 @@ Example usage:
         [pytest]
         postgresql_port = 8888
 
+Maintaining database state outside of the fixtures
+--------------------------------------------------
+
+It is possible and appears it's used in other libraries for tests,
+to maintain database state with the use of the ``pytest-postgresql`` internals
```

Comment:
I think it's not internals now.

It's about not using fixtures (so no dependency on pytest?) and having something else start/stop PostgreSQL (e.g. running it in a separate container)?

Reply:
Actually few pytest based projects are using pytest-postgresql just for these database managing functions

Reply:
And yes, containers (ie Warehouse)